docs/examples/applied_freeze.py

```diff
@@ -71,12 +73,21 @@ def _preprocess_bb_cira_forecast_dataset(ds: xr.Dataset) -> xr.Dataset:
     preprocess=_preprocess_bb_cira_forecast_dataset,
 )
 
+# Load the climatology for DurationME
+climatology = xr.open_zarr(
+    "gs://extremeweatherbench/datasets/surface_air_temperature_1990_2019_climatology.zarr",  # noqa: E501
+    storage_options={"anon": True},
+    chunks="auto",
+)
+climatology = climatology["2m_temperature"].sel(quantile=0.15)
+
+# Define the metrics
 metrics_list = [
-    metrics.RMSE(),
-    metrics.MinimumMAE(),
-    metrics.OnsetME(),
-    metrics.DurationME(),
+    metrics.RMSE,
+    metrics.MinimumMAE,
+    metrics.DurationME(criteria=climatology, op_func=operator.le),
```

Comment:
This is a little unergonomic for average scientific Python users. Let's add a utility somewhere that simplify defines a dictionary mapping the supported operators to their binary funcs, e.g.

``` python
operators = {
    ">": operator.gt,
    ">=": operator.ge,
    "<": operator.lt,
    "<=": operator.le
}
```